### DISCUSSION TITLE
홈에서 페이지 번호 노출

### DISCUSSION THREAD

**Discussion by teng-ny** [pronist/hELLO#219] [Q&A] [answered]
홈을 선택했을 때 페이지 번호를 노출시키고 싶은데 1페이지만 선택되어있더라구요!   
이를 변경할 수 있을까요?  

<전>  
<img width="1427" alt="스크린샷 2023-06-21 오후 2 37 01" src="https://github.com/pronist/tistory-skin-hELLO/assets/96412996/20906526-8186-4b00-92ec-89d653975c14">  
  
<후>   
<img width="1427" alt="스크린샷 2023-06-21 오후 2 36 03" src="https://github.com/pronist/tistory-skin-hELLO/assets/96412996/568e316d-3f5f-48b7-baaf-f17307f0502e">

**Comment by pronist** [comment, marked as answer]
@teng-ny 안녕하세요. **페이지에 따른 페이징의 노출 여부는 스킨 제작자가 아닌 티스토리가 정합니다.** 즉, 스킨 개발자는 페이징을 어떤 위치놓겠다는 정도만 지정할 뿐 페이지에 따른 개별적인 노출 여부는 결정할 수 없습니다. 따라서 수정해드릴 수 있는 개선점이 없습니다. 이 점 양해 부탁드립니다.
